EU4toV2/Source/EU4World/Provinces/EU4Province.h

```diff
@@ -1,17 +1,16 @@
 #ifndef EU4_PROVINCE_H
 #define EU4_PROVINCE_H
-
 #include "Date.h"
 #include "BuildingWeightEffects.h"
 #include "PopRatio.h"
 #include "ProvinceBuildings.h"
 #include "ProvinceHistory.h"
 #include "ProvinceStats.h"
-#include "../../Mappers/Buildings/Buildings.h"
-#include "../Modifiers/Modifiers.h"
+#include "Buildings/Buildings.h"
+#include "Modifiers/Modifiers.h"
 #include "Parser.h"
-#include "../../Mappers/SuperGroupMapper/SuperGroupMapper.h"
-#include "../Regions/Regions.h"
+#include "SuperGroupMapper/SuperGroupMapper.h"
+#include "Regions/Regions.h"
```

Comment:
Many of the pre-existing functions are missing tests. Even for the simple ones, tests should be added as a demonstration of use.